### PR TITLE
AMBARI-24956. Log Search: cleanup audit/service log config name prefixes

### PR DESCRIPTION
**Pull request by oleewere**
# What changes were proposed in this pull request?
Cleanup logsearch property names - use similar prefixes, do not mix it up
## How was this patch tested?
with docker env

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/80/
Test PASSed.

File: docker/test-config/logsearch/logsearch-sso.properties [hunk 1]
```diff
@@ -16,20 +16,16 @@
...
-logsearch.service.logs.split.interval.mins=15
-logsearch.collection.service.logs.numshards=3
-logsearch.collection.service.logs.replication.factor=2
+logsearch.solr.service.logs.collection=hadoop_logs
+logsearch.solr.service.logs.numshards=3
+logsearch.solr.service.logs.replication.factor=2
 
 # Audit logs
 logsearch.solr.audit.logs.zk_connect_string=localhost:9983
 logsearch.solr.collection.audit.logs=audit_logs
```
Comment (adoroszlai) [inline docker/test-config/logsearch/logsearch-sso.properties]: Should be changed to `logsearch.solr.audit.logs.collection`?

**Comment by swagle** [comment]
No upgrade catalog changes needed?

**Comment by oleewere** [comment]
@swagle , yes it will be needed, but that one is for ambari-server side

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-LogSearch-Github-PR-Builder/82/
Test PASSed.

**Pull request merged**